src/main/java/org/traccar/config/Keys.java

```diff
@@ -687,6 +687,28 @@ private Keys() {
             "sms.http.template",
             Collections.singletonList(KeyType.GLOBAL));
 
+    /**
+     * AWS Access Key with SNS permission.
+     */
+    public static final ConfigKey<String> AWS_ACCESS_KEY = new ConfigKey<>(
```

Comment:
Please make sure variable names match keys. For example this should be `SMS_AWS_ACCESS`.